### PR TITLE
Community: Add section to discourage 1:1 support queries

### PR DESCRIPTION
**Pull request by beorn7**
I find myself quite often in the situation where I have to tell people
what I'm adding here. Even if not all will read it, it is then easy to
point people to it.

@brian-brazil @fabxc @juliusv Does this make sense?

### CODE REVIEW

**Comment by brian-brazil** [comment]
:+1: 

I try to convert them into customers, but that's not for everyone :)


Do we also want to add something about not using the issue tracker for support requests? That has been becoming more common, distracting from actual issues.

**Comment by beorn7** [comment]
Yes, I thought so, too. Issue tracker for bugs, mailing list for questions.

File: content/community.html [hunk 1]
```diff
@@ -27,6 +27,14 @@
     <p>
       <strong>Issue tracker:</strong> We use the GitHub issue tracker for the various <a href="http://github.com/prometheus">Prometheus repositories</a>
     </p>
+    <p>
+      <em>Please refrain from soliciting individual project members
```
Comment (matthiasr) [inline content/community.html]: Maybe use use less fancy language here? "do not ask"?

File: content/community.html [hunk 2]
```diff
@@ -27,6 +27,14 @@
...
+      for support. Use the channels above instead, where you will find
+      more potential helpers, and the whole community can benefit from
```
Comment (matthiasr) [inline content/community.html]: "where the whole community can help you and benefit from the solutions provided"

Comment (beorn7) [inline content/community.html]: I like it.

File: content/community.html (hunk 1 above)
Comment (beorn7) [inline content/community.html]: Makes sense.

**Comment by beorn7** [comment]
I'll work in MR's suggestions, but wait for @fabxc and/or @juliusv to get quorum.

**Comment by juliusv** [comment]
:+1: to everything, including MR's comments.

**Pull request merged**